kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/IcebergSinkConfig.java

```diff
@@ -406,6 +406,10 @@ public JsonConverter jsonConverter() {
     return jsonConverter;
   }
 
+  static boolean checkClassName(String className) {
```

Comment:
One more nit, could you add `@VisibleForTesting`? That makes it clear why it isn't private.